### PR TITLE
fix: de-duplicate metadata submission after one minute

### LINKED ISSUE #340: Checksum Matches Existing Object - Json submit endpoint
When trying to re-submit a JSON that was previously deleted from production, I encounter the error:

`detail": "provided checksum matches existing object: {'checksums.type': 'sha256', 'checksums.checksum': '9eaece59f58b4d58008f2efcf556736c0f5a0cbe16311206d7f6b127e9cfc580'}`

### PR DESCRIPTION
**Pull request by dwinston**
re-submission of "same" changes is a valid use case

closes #340

### CODE REVIEW

**Review by eecavanna — APPROVED** [review]
Looks like this is to guarantee that the `base` value changes at least once per minute. I don't know the context here, but the code change looks straightforward to me.

**Comment by eecavanna** [comment]
I recommend holding off on merging this until there is no risk of such a merge complicating the resolution of issue https://github.com/microbiomedata/nmdc-runtime/issues/348.

**Review by PeopleMakeCulture — APPROVED** [review]

**Pull request merged**
